hrm/department/models/models.py

```diff
@@ -1 +1,24 @@
 """Model declaration"""
+from django.db import models
+
+
+class Department(models.Model):
```

Comment:
The only thing that I'd change is the primary key field. Lest define it directly to suppress warnings when starting application server